code/go/0chain.net/blobbercore/handler/handler_common.go

```diff
@@ -133,7 +134,9 @@ func WithStatusConnectionForWM(handler common.StatusCodeResponderF) common.Statu
 		}
 		mutex.Lock()
 		defer mutex.Unlock()
-		ctx = GetMetaDataStore().CreateTransaction(ctx)
+		ctx = GetMetaDataStore().CreateTransaction(ctx, &sql.TxOptions{
+			Isolation: sql.LevelRepeatableRead,
```

Comment:
what is the reason to change it to repeatable read?

Reply:
Originally did it so that results of move file store are reflected, but realized they are executed in a different transaction so won't matter. I will remove the change